i18n/extract.py

```diff
@@ -179,6 +179,16 @@ def run(self, args):
         self.rename_source_file('django-saved.po', 'django.po')
         self.rename_source_file('djangojs-saved.po', 'djangojs.po')
 
+        # Remove the POT-Creation-Date from the all files.
+        for filename in files_to_clean.union({
+            'django.po',
+            'djangojs.po',
+        }):
+            LOG.info('Removing [POT-Creation-Date] from %s', filename)
+            profile = polib.pofile(self.source_msgs_dir.joinpath(filename))
+            profile.metadata.pop('POT-Creation-Date', None)
+            profile.save()
```

Comment:
nitpick: Maybe we should remove the date only if there's a date in the first place:

I'm not sure if this has a practical difference since the date might be always present.

```suggestion
            if 'POT-Creation-Date' in profile.metadata:
                LOG.info('Removing [POT-Creation-Date] from %s', filename)
                profile = polib.pofile(self.source_msgs_dir.joinpath(filename))
                profile.metadata.pop('POT-Creation-Date')
                profile.save()
```